src/extensions/extension-discovery.ts

```diff
@@ -365,7 +365,13 @@ export class ExtensionDiscovery extends Singleton {
       let isCompatible = isBundled;
 
       if (manifest.engines?.lens) {
```

Comment:
Is this code actually relevant to the bundled extensions? If so should we skip this if the extension is bundled?
```
if (!isBundled && manifest.engines?.lens) {
```

Reply:
You are right, we could skip the check for bundled extensions.